### ISSUE TITLE
<FormField/> - Label in is not accessible 

### ISSUE THREAD

**Issue by rudnitskih** [wix/wix-style-react#3258]
# 🐛 Bug Report

### 🏗 Relevant Components

`FormField`

### 😯 Current Behavior

`label` is not connected to the input element. It causes that screen readers cannot help users to understand what input means. Also, it breaks behavior when the user clicks label - input gets focus.

Screenshot from WSR storybook:
![image](https://user-images.githubusercontent.com/8848858/56020802-aa53f680-5d10-11e9-993c-d13fb32655c5.png)

### 🤔 Expected Behavior

Input should be wrapped by label or at least they should be wired by unique ID tag.

### 👣 Steps to Reproduce

https://wix-wix-style-react.surge.sh/?selectedKind=Components&selectedStory=FormField&full=0&addons=0&stories=1&panelRight=0

  - **WSR version**: `6.13.0`

### 👀 Severity

- Major

**Comment by shlomitc** [comment]
I guess this is what you’re looking for
https://github.com/wix/wix-style-react/blob/master/src/FormField/FormField.js#L78-L86
Maybe the right name for it should be htmlFor like the native label, but it does what you need

**Comment by rudnitskih** [comment]
@shlomitc I think it is important to have good defaults here. I am not sure that users of WSR know about this issue. And at the same time, it breaks basic web form implementation. Maybe we should think about automatic solution how we can generate unique ID under the hood?

**Comment by shlomitc** [comment]
Creating built-in `id`s to reusable components is a bad practice as you never know if a previous id was created in an app. It can also be an issue in SSR and other cases. This is why it is the consumer's responsibility.

**Comment by EgoziE** [comment]
Opened a task in Jira creating a proper example to show how to create a form end to end - closing this issue.
